pkg/cloud/gcp/actuators/machine/machine_scope_test.go

```diff
@@ -8,11 +8,12 @@ import (
 	"testing"
 	"time"
 
+	"github.com/openshift/machine-api-provider-gcp/pkg/apis/gcpprovider/v1beta1"
```

Comment:
Use "github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/util", and include in the rest of the imports please